align_system/algorithms/comparative_regression_adm_component.py

```diff
@@ -136,10 +168,47 @@ def run(self,
                         attribute_prediction_reasonings[choice].setdefault(
                             attribute.kdma, []).append(response[choice]['reasoning'])
                     # Probe level reasoning
-                    except:
+                    except KeyError:
```

Comment:
I'm assuming this is the error we want to `except` here, but someone correct me if I'm wrong (always better to be as specific as possible when using `except`)